index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <title>Artificial intelligence forum</title>
+  </head>
+  <body>
+    <!-- popup menu area -->
+
+    <div id="close" class="js-menu hide show2">
+      <div class="menu-wrapper">
+        <a id="cancel" class="hide hide2"
+          ><img class="canceli" src="IMG/Icon-Cancel.png" alt="icon"
+        /></a>
+        <img class="logo-nav logo-nav1" src="./IMG/itu-logo.png" alt="logo" />
+        <ul class="js-list" id="mobile-menu-list">
+          <li>
+            <a href="#portfolio" class="menu-list" id="elem1">UTI Projects</a>
+          </li>
+          <li>
+            <a href="./about-page.html" class="menu-list" id="elem2">About</a>
+          </li>
+          <li>
+            <a href="#contact-me" class="menu-list nav-styled" id="elem3">Partners</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <!-- popup menu area -->
+
+    <header class="header-wrapper">
+      <div class="menu-icon hide2" id="open"><i class="fa fa-reorder"></i></div>
+    </header>
+    <main>
+      <section class="middle-section-wrapper">
+        <section class="header-resizing">
+          <h3 class="greetings">"Hello!smart world"</h3>
+          <h1 class="top-header">GLOBAL FORUM ON AI AND IoT</h1>
+        </section>
+        <section class="text-container">
+          <p class="text1">
+            A platform to discuss trends in Artificial Intelligence, Internet of
+            Things and Smart Cities and discuss the role of artificial
+            intelligence and IoT to shape smarter and more sustainable cities
+            and to address urban challenges, with the final goal of improving
+            the quality of life of people.
```

Comment:
- [ ] Kindly make sure to follow the colour as per the original design for the heading. Use #ec5242.
- [ ] Kindly make sure that the hamburger is of #272a31 colour as per the original design.
- Original design:
<img width="392" alt="Screenshot 2022-04-15 at 3 40 01 PM" src="https://user-images.githubusercontent.com/70269507/163571953-0d46c424-1903-4290-b0de-e5e9c382bd06.png">

- Your design:
<img width="501" alt="Screenshot 2022-04-15 at 3 39 09 PM" src="https://user-images.githubusercontent.com/70269507/163571957-a58d0184-2910-4701-9f1c-0bb4df2e61e2.png">